presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -1141,7 +1141,7 @@ public void testViewFunctionAccessControl()
 
         assertAccessDenied(
                 "SELECT * FROM test_view_function_access",
-                "'test_view_access_owner' cannot grant 'abs' execution to user 'hive'",
+                "'test_view_access_owner' cannot grant 'abs' execution to user '.*'",
```

Comment:
`.*` or `\\w*` -- can we assume username is just alphanumeric?

Reply:
I think so.